test/Microsoft.AspNet.Razor.Test/TagHelpers/TagHelperParseTreeRewriterTest.cs

```diff
@@ -19,6 +19,204 @@ namespace Microsoft.AspNet.Razor.Test.TagHelpers
 {
     public class TagHelperParseTreeRewriterTest : CsHtmlMarkupParserTestBase
     {
+        public static TheoryData EmptyTagHelperBoundAttributeData
+        {
+            get
+            {
+                var factory = CreateDefaultSpanFactory();
+                var emptyAttributeError =
+                    "Attribute '{0}' on tag helper element '{1}' is a '{2}' tag helper bound attribute with no " +
+                    "value. Only '{3}' typed attributes can be without a value.";
+                var boolTypeName = typeof(bool).FullName;
+                var stringTypeName = typeof(string).FullName;
+
+                // documentContent, expectedOutput, expectedErrors
+                return new TheoryData<string, MarkupBlock, RazorError[]>
+                {
+                    {
+                        "<myth bound='' />",
```

Comment:
also test `<myth bound='  ' />`.  I think this currently results in "success" though the generated code will always be erroneous.